BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift

```diff
@@ -19,6 +19,14 @@ public struct MetadataOverrides {
     public var applicationName: String?
     public var custom: [String: Any]?
     public var duration: Int?
+    // swiftlint:disable line_length
```

Comment:
`line_length` violations should be turned off for URLs 🤔 You should be able to remove the `// swiftlint:disable line_length` and `// swiftlint:enable line_length` for this

Reply:
Unfortunately the violations are reported, so I'm keeping the disable comments for now.